blocks/blockstore/blockstore.go

```diff
@@ -74,6 +74,10 @@ type blockstore struct {
 }
 
 func (bs *blockstore) Get(k key.Key) (blocks.Block, error) {
+	if k == "" {
+		return nil, ErrNotFound
```

Comment:
Yeah i thought about that too, but we'd be introducing another error type, to watch out for. not sure where else errors are checked explicitly, and seemed that treating "" as an "always not found" key was ok.